src/EnergyPlus/WindowManager.cc

```diff
@@ -1887,6 +1887,7 @@ namespace WindowManager {
         for (ConstrNum = 1; ConstrNum <= TotConstructs; ++ConstrNum) {
             if (Construct(ConstrNum).FromWindow5DataFile) continue;
             if (Construct(ConstrNum).WindowTypeBSDF) continue;
+            if (Construct(ConstrNum).TypeIsAirBoundaryInteriorWindow) continue;
```

Comment:
This is the main reason InteriorWindow wasn't working - this was zeroing out all of the window contruction properties that had been set before.

Reply:
Sneaky